src/evaluator.ts

```diff
@@ -828,14 +877,14 @@ class Evaluator {
     // Run the evals
     if (this.options.interactiveProviders) {
       runEvalOptions = runEvalOptions.sort((a, b) =>
-        a.provider.id().localeCompare(b.provider.id()),
+        a[0].provider.id().localeCompare(b[0].provider.id()),
```

Comment:
All evals in a thread must have the same provider. It should therefore be safe to look at `[0]` here.